static/src/stylesheets/module/facia/_slices.scss

```diff
@@ -171,8 +171,8 @@ Hence why a greater depth of selector specificity is needed.
         display: none;
     }
 
-    .fc-item__live-indicator .live-pulse-icon {
-        display: none;
+    .live-pulse-icon:before {
+        content: none;
```

Comment:
Yes I think it's a good thing to do. I want this to deploy as a quick fix but I will move it to one file in another PR.